cookbook/docs/index.md

```diff
@@ -69,6 +69,12 @@ First install [flytekit](https://pypi.org/project/flytekit/), Flyte's Python SDK
 pip install flytekit
 ```
 
+The [sklearn](https://scikit-learn.org/stable) package is a secondary dependency for this demo.
```

Comment:
let's collapse this into the above `pip install`, adding a sentence there about sklearn being a dependency, something like:

```
First install [flytekit](https://pypi.org/project/flytekit/), Flyte's Python SDK, and sklearn, which we'll need for this guide.
```

Reply:
done, lmk if the updated doc works